webcompat/templates/layout.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>webcompat.com</title>
+  <title>{% block title %}{% endblock %}webcompat.com</title>
```

Comment:
All your templates for titles could be probably more compact and less prone to errors if you put the pipe here.

So if you do 
```
<title>{% block title %}{% endblock %} | webcompat.com</title>
```

Then your templates can become: 

```
{% block title %}Issue #{{ number }}{% endblock %}
```